packages/elements/src/photon-med-search/photon-med-search-component.tsx

```diff
@@ -77,7 +77,7 @@ customElement(
 
     onMount(async () => {
       const { data } = await client!.apollo.query({ query: GET_CATALOGS });
-      if (data.catalogs.length > 0) {
+      if (data?.catalogs.length > 0) {
```

Comment:
Does `onMount` get called more than once?

Reply:
There are a few possibilities of whats happening here:
- the query occasionally fails. In which case we should investigate why
- this component hits the `onMount` lifecycle multiple times, in which case we should investigate why.
- There might be some other ways

Reply:
@marisbest2 I'm not seeing multiple onmounts